src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+import PropTypes from "prop-types";
+import PostHeader from './PostHeader';
+import LikesContainer from './LikesContainer';
+import CommentSection from '../CommentSection/CommentSection';
+import styled from 'styled-components';
+import './PostContainer.css';
+
+// Styled Components
+const PostWrapper = styled.div`
+    border: 1px solid gray;
+    width: 800px;
+    margin: auto;
+    margin-bottom: 40px;
+`;
+
+const PostImgWrapper = styled.img`
+    width: 798px;
+    height: 798px;
+`;
+
+// Body of the container
+
+class PostContainer extends Component {
+    // Props taken in:
```

Comment:
Nice visualization - I can see what you are receiving and how you are utilizing them!